improver/psychrometric_calculations/hail_size.py

```diff
@@ -299,7 +283,7 @@ def extract_pressure_at_268(self, temperature_on_pressure: Cube) -> (CubeList):
         coord_order = [coord.name() for coord in temperature_on_pressure.coords()]
         coord_order.remove("pressure")
 
-        re_ordered_temperature = self.coordinate_order([temperature_on_pressure])[0]
+        [re_ordered_temperature] = self.coordinate_order([temperature_on_pressure])
```

Comment:
This can be achieved with a magical comma. It will raise an exception it more than one thing is returned.
```suggestion
        re_ordered_temperature, = self.coordinate_order([temperature_on_pressure])
```